tests/btf/assembly/exported-symbols.rs

```diff
@@ -1,5 +1,5 @@
 // assembly-output: bpf-linker
-// compile-flags: --crate-type bin -C link-arg=--emit=obj -C debuginfo=2 -C link-arg=--log-level=debug -C link-arg=--log-file=/tmp/linker-lol.log
+// compile-flags: --crate-type bin -C link-arg=--emit=obj -C debuginfo=2 -C link-arg=--log-level=debug -C link-arg=--log-file=/tmp/linker-lol.log -C link-arg=--btf
```

Comment:
Oops can you please remove --log-level and --log-file here? I didn't mean to commit those 😅

Reply:
done